incubator/eventrouter/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
@unguiculus updated the examples in the Helm docs, could you maybe update this file to use one from one of the existing charts in this repo?